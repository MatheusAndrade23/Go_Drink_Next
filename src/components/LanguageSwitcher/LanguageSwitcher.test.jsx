import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { LanguageSwitcher } from '.';

describe('<LanguageSwitcher />', () => {
  it('should render', () => {
    const { container } = renderTheme(<LanguageSwitcher />);
    expect(container).toMatchSnapshot();
  });
});
