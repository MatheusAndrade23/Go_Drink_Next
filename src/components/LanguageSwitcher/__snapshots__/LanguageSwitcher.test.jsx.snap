// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LanguageSwitcher /> should render 1`] = `
.c0 {
  position: fixed;
  background-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center !important;
  -webkit-justify-content: center !important;
  -ms-flex-pack: center !important;
  justify-content: center !important;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #EEEEEE;
  bottom: 35px;
  right: 20px;
  z-index: 6;
  padding: 4px 0;
  font-weight: bold;
}

.c0 p {
  opacity: 0.5;
  margin-right: 10px;
}

.c1 {
  cursor: pointer;
  background: transparent;
  border: none;
  margin-left: 5px;
  opacity: 0.5;
}

.c1 span {
  font-size: 20px;
}

<div>
  <div
    class="c0"
  >
    <p>
      language
    </p>
    <button
      class="c1"
      name="Português"
      title="Português"
    >
      <span
        class="fi fi-br fis"
      />
    </button>
    <button
      class="c1"
      name="English"
      title="English"
    >
      <span
        class="fi fi-us fis"
      />
    </button>
  </div>
</div>
`;
